### PR TITLE
User guide configuration for events

### PR DESCRIPTION
**Pull request by peterbryntesson**
User guide for configuring for events

### CODE REVIEW

File: docs/opc-event-configuration.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+Use configuration files for OPC UA Events in OPC Publisher
```
Comment (simonjaeger) [inline docs/opc-event-configuration.md:1]: This you can make a header.

**Review by simonjaeger — APPROVED** [review]
LGTM

**Comment by koepalex** [comment]
Nice write, is there also another doc planned with the generated output formats? If not I would add them to the description

**Review by koepalex — CHANGES_REQUESTED** [review]
:clock1:

**Review by koepalex — APPROVED** [review]
:shipit:

**Comment by peterbryntesson** [comment]
> Nice write, is there also another doc planned with the generated output formats? If not I would add them to the description

Yes, I will write that in a separate doc later this week.

**Pull request merged**